addons/clones/spriteCount.js

```diff
@@ -0,0 +1,67 @@
+export default async function ({ addon, console }) {
+  const vm = addon.tab.traps.vm;
+  const ogCloneCounter = vm.runtime.changeCloneCounter;
+
+  async function updateCounts() {
+    let counts = {};
+    let remove = addon.self.disabled || !addon.settings.get("showSpriteCount");
+
+    vm.runtime.targets
+      .filter((target) => !target.isOriginal)
+      .map((target) => target.sprite.name)
+      .forEach((target) => {
+        if (!counts[target]) counts[target] = 1;
+        else counts[target] += 1;
+      });
+
+    const spriteWrapper = await addon.tab.waitForElement("[class*=sprite-selector_items-wrapper]");
+    const spriteNames = Array.from(spriteWrapper.querySelectorAll("[class*=sprite-selector-item_sprite-name]"));
+
+    spriteNames.forEach((spriteName) => {
+      if (counts[spriteName.innerText.split("\n")[0]] !== undefined) {
+        const existingElement = spriteName.querySelector(".sa-clone-count");
+        let count;
+
+        if (remove) {
+          existingElement?.remove();
```

Comment:
can we hide instead of remove? its always faster to show/hide dom rather than delete/recreate.

Reply:
remove is only used for when the addon is disabled, but yeah ill look into it